app/views/layouts/_footer.html.haml

```diff
@@ -0,0 +1,29 @@
+%br
+  .page-footer.blue.center-on-small-only.bg-info
+    %br
+    .container-fluid
+      .row
+        .col-md-6
+          .title.text-white
+            EthermineApp
+            %p{class: 'text-white'} With this awseome app you can get the information about a miner
+            .title.text-white
+              You can get Addresses From Here:
+            %ul
+              %li
+                %a{href: "https://ethermine.org/", class: 'text-white'}= "Ethermine.org"
+        .col-md-6
+          .title.text-white
+            Links:
+          %ul
+            %li
+              %a{href: "https://rubygems.org/gems/etherclient", class: 'text-white'}= "Etherclient gem"
```

Comment:
wth? Are you using rails to add a string into an html anchor?

Reply:
fix all the stuff like this `= "Etherclient gem"`

Reply:
ok, thanks for the tip